applications/fishing-map/src/features/analysis/Analysis.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useRef, useMemo, useState, Fragment } from 'react'
 import cx from 'classnames'
+import * as ua from 'react-ga'
```

Comment:
Is there any specific reason to import the entire package ? We might can see some bundle size if importing just what we need.

Reply:
Ok, I'll change it to import just the methods that are used